README.md

```diff
@@ -459,19 +468,7 @@ Fields listed under `task_definition` correspond to fields that will be included
   * In Docker compose version 2, the `cpu_shares`, `mem_limit`, and `mem_reservation` fields can be specified in either the compose or ECS params file. If they are specified in the ECS params file, the values will override values present in the compose file.
   * If you are using a private repository for pulling images, `repository_credentials` allows you to specify an AWS Secrets Manager secret ARN for the name of the secret containing your private repository credentials as a `credential_parameter`.
 
-Example `ecs-params.yml` with service resources specified:
```

Comment:
since you're removing this services example, can you add another sub-bullet under `services` that says something like: "See sample ecs-params.yml below for example uses."

Reply:
Is that really needed? The way it reads right now is that we have the schema, then we have explanations, then we have a bunch of examples. I think that makes complete sense and there doesn't need to be a special bullet under services noting that there are examples for this field.

Reply:
If this is the case of all schema field examples (and not just that you moved or omitted the service one) than I agree it's fine as is.